src/compas/scene/group.py

```diff
@@ -0,0 +1,37 @@
+from compas.data import Data
+
+from .sceneobject import SceneObject
+
+
+class Group(SceneObject):
+    """A group of scene objects.
+
+    Parameters
+    ----------
+    name : str, optional
+        The name of the group.
+
+    Examples
+    --------
+    >>> from compas.scene import Scene
+    >>> from compas.scene import Group
+    >>> from compas.geometry import Point
+    >>> scene = Scene()
+    >>> group = Group(name="My Group")
+    >>> scene.add(group)
+    >>> point = Point(0, 0, 0)
+    >>> group.add(point)
+    >>> print(scene)
+    <Tree with 3 nodes>
+    └── <TreeNode: ROOT>
+        └── <Group: My Group>
+            └── <GeometryObject: Point>
+
+    """
+
+    def __new__(cls, *args, **kwargs):
+        return object.__new__(cls)
```

Comment:
We need to overwrite `____new____` to revert to the default behavior of normal `object`, so I can initialize a instance directly without looking for a registered item.

Reply:
I think this comment will be needed in code, otherwise we will not remember why this is like this.

Reply:
Done!